Dockerfile

```diff
@@ -0,0 +1,16 @@
+# syntax=docker/dockerfile:1
+FROM ruby:2.6.3
```

Comment:
do tests run with this image? I think it does not include browsers so the feature tests will fail